github/search.go

```diff
@@ -15,6 +15,13 @@ import (
 // SearchService provides access to the search related functions
 // in the GitHub API.
 //
+// Each method takes a query string defining the search keywords and any search qualifiers. For example,
+// when searching issues, the query "gopher is:issue language:go" will search for issues containing the word "gopher"
```

Comment:
Mind wrapping these lines at around 90 characters for better readability? It's not related to terminal widths, just that really long lines are harder to read/scan with eyes.

Reply:
Also, since the code below does `cl.Search.Issues(ctx, "gopher is:issue label:bug language:go", opts)`, I think you should use the same search query in the text here. Otherwise it might be confusing. It's currently missing `label:bug`. Either add it here, or remove it in the code below.